kratos/tests/cpp_tests/utilities/test_math_utils.cpp

```diff
@@ -640,23 +640,20 @@ namespace Kratos
         KRATOS_TEST_CASE_IN_SUITE(MathUtilsMatrixSquareRoot, KratosCoreFastSuite)
         {
             // Input matrix
-            Matrix mat66(6,6);
-            mat66(0,0) = 6.77; mat66(0,1) = 2.52; mat66(0,2) = 1.98; mat66(0,3) = 0; mat66(0,4) = 0; mat66(0,5) = 4.44;
-            mat66(1,0) = 2.52; mat66(1,1) = 3.92; mat66(1,2) = 1.96; mat66(1,3) = 0; mat66(1,4) = 0; mat66(1,5) =-3.00;
-            mat66(2,0) = 1.98; mat66(2,1) = 1.96; mat66(2,2) = 5.10; mat66(2,3) = 0; mat66(2,4) = 0; mat66(2,5) = 9.90;
-            mat66(3,0) = 0; mat66(3,1) = 0; mat66(3,2) = 0; mat66(3,3) = 1.98; mat66(3,4) = 9.78; mat66(3,5) = 0;
-            mat66(4,0) = 0; mat66(4,1) = 0; mat66(4,2) = 0; mat66(4,3) = 9.78; mat66(4,4) = 2.17; mat66(4,5) = 0;
-            mat66(5,0) = 4.44; mat66(5,1) = -3.00; mat66(5,2) = 9.90; mat66(5,3) = 0; mat66(5,4) = 0; mat66(5,5) = 2.57;
```

Comment:
How was this working before?

Reply:
OK. I guess it should be related with the `nan` check. Am I wrong?

Reply:
Yes, it was because the NaN

Reply:
Reapproval?